duckscript_sdk/src/sdk/std/fs/rm/mod_test.rs

```diff
@@ -56,6 +56,22 @@ fn run_path_is_file() {
     assert!(!path.exists());
 }
 
+#[test]
+fn run_path_is_file_and_with_flag() {
+    let path = Path::new("./target/_duckscript/rm/file.txt");
```

Comment:
can we put a unique file name here so tests running in parallel won't conflict?

Reply:
Should be that, I'll change it right away